libraries/SITL/SIM_BattMonitor_SMBus.cpp

```diff
@@ -10,19 +10,31 @@ SITL::SIM_BattMonitor_SMBus::SIM_BattMonitor_SMBus() :
     add_register("Current", SMBusBattDevReg::CURRENT, O_RDONLY);
     add_register("Remaining Capacity", SMBusBattDevReg::REMAINING_CAPACITY, O_RDONLY);
     add_register("Full Charge Capacity", SMBusBattDevReg::FULL_CHARGE_CAPACITY, O_RDONLY);
+    add_register("Design Charge Capacity", SMBusBattDevReg::DESIGN_CAPACITY, O_RDONLY);
+    add_register("Design Maximum Voltage", SMBusBattDevReg::DESIGN_VOLTAGE, O_RDONLY);
     add_register("Cycle_Count", SMBusBattDevReg::CYCLE_COUNT, O_RDONLY);
     add_register("Specification Info", SMBusBattDevReg::SPECIFICATION_INFO, O_RDONLY);
     add_register("Serial", SMBusBattDevReg::SERIAL, O_RDONLY);
     add_block("Manufacture Name", SMBusBattDevReg::MANUFACTURE_NAME, O_RDONLY);
     add_block("Device Name", SMBusBattDevReg::DEVICE_NAME, O_RDONLY);
     add_register("Manufacture Data", SMBusBattDevReg::MANUFACTURE_DATA, O_RDONLY);
+    add_block("Device Chemistry", SMBusBattDevReg::DEVICE_CHEMISTRY, O_RDONLY);
 
     set_register(SMBusBattDevReg::TEMP, (int16_t)((15 + C_TO_KELVIN)*10));
      // see update for voltage
      // see update for current
-     // TODO: remaining capacity
-     // TODO: full capacity
-     set_register(SMBusBattDevReg::CYCLE_COUNT, (uint16_t(39U)));
+
+    set_register(SMBusBattDevReg::DESIGN_VOLTAGE, (uint16_t(50000U)));          // (mV) Design maximum voltage
```

Comment:
indent wrong

Reply:
The indents here were originally 5 spaces. I fixed them to 4 spaces as a separate commit to match mine.